serverless.yml

```diff
@@ -673,3 +673,31 @@ functions:
               - name: request.querystring.limit
               - name: request.querystring.offset
               - name: request.header.origin
+
+  network_statistics_handler:
+    handler: handlers/network_statistics.get_basic_statistics
+    timeout: 5
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/network_statistics.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: network-statistics
+          method: get
+          cors:
+            headers:
+              - Content-Type
+              - Cache-Control
+            methods:
+              - GET
+              - OPTION
```

Comment:
Lambda handles the OPTION request automatically? Does that count as a lambda call?

Reply:
Oh! This is a configuration for the API Gateway. But I need to test it after the deploy.

Reply:
Is it really needed? I thought this was implied since we are enabling CORS.
 If so, shouldn't it be `OPTIONS`?

Reply:
Besides, you can test this in our dev environment before merging this PR. I can show you how.
This way we won't have to create new PRs for fixes.